redaxo/src/addons/users/pages/users.php

```diff
@@ -112,6 +112,10 @@
         }
         $warnings[] = $msg;
     }
+
+    if (!rex::getUser()->isAdmin() && $user->isAdmin()) {
+        $warnings[] = rex_i18n::msg('user_requires_admin_rights');
```

Comment:
Ich wundere mich, dass man hierher kommt. Wird das nicht eigentlich versucht schon in Zeile 87 zu verhindern.
Kapiere gerade noch nicht, wieso die Zeile 87 nicht greift. Kapierst du es?

Allgemein würde ich eigentlich noch früher ansetzen, und die Edit-Maske gar nicht erst anzeigen.
Würde stattdessen die Liste der User anzeigen, und da dann vielleicht den Fehler.
Alternativ wäre auch harte Exception für mich ok, denn normal kommt man ja gar nicht zu der Stelle, die Admins sind ja nicht verlinkt in der Liste für Nicht-Admins.

Reply:
in Zeile 87 wird `$sql` benutzt, was nur initialisiert wird und somit `!$sql->getValue('admin')` immer `true` war.

Reply:
mal schauen ob wir diesen fehler mit rexstan erkennen bzw. erkennen können:
https://github.com/FriendsOfREDAXO/rexstan/issues/177